CONTRIBUTING.md

```diff
@@ -0,0 +1,157 @@
+# Contributing to Innovaccer Design System
+
+We are happy that you wish to contribute to this project. For that reason, we
+present you with this guide.
+
+## Contents
+
+- [Contents](#contents)
+- [How Do I Contribute?](#how-do-i-contribute)
+- [Development](#development)
+- [Reporting Bugs](#reporting-bugs)
+    - [Issue Search](#issue-search)
+    - [Check If It's Been Fixed](#check-if-its-been-fixed)
+- [Request Features](#request-features)
+    - [Submitting a Pull Request](#submitting-a-pull-request)
+    - [Make Changes and Commit](#make-changes-and-commit)
+- [Update Tests](#update-tests)
+
+## How Do I Contribute?
+
+There are different ways to contribute, each with a different level
+of involvement and technical knowledge required, such as:
+
+* [Reporting Bugs](#reporting-bugs)
+* [Request Features](#request-features)
+
+**Please read this document carefully. It will help maintainers and readers
+in solving your issue(s), evaluating your feature request, etc.**
+
+## Development
+
+```bash
+#clone repository
+git clone https://github.com/innovaccer/design-system.git
+
+#install dependencies
+npm install
+
+#start development server
+npm run dev
+```
+
+The server runs on a random port and url is opened automaticaly in browser as the project uses storybook for component developement and documentation.
+
+
+
+## Reporting Bugs
+
+We welcome clear, detailed bug reports.
+
+**Bugs are considered features that are not working as described in
+documentation.**
+
+If you've found a bug, please file a report in our [issue tracker](https://github.com/innovaccer/design-system/issues).
+
+
+### Issue Search
+
+Search to see if it has already been reported via
+the issue search.
+If so, up-vote it (using GitHub reactions) or add additional helpful details to
+the existing issue to show that it's affecting multiple people.
+ 
+### Check If It's Been Fixed
+
+Check if the issue has been fixed — try to reproduce it using the latest
+`master` or development branch in the repository.
+
+## Request Features
+
+New feature requests are welcome. Analyse whether the idea fits within scope of
+the project. Then, detail your request, ensuring context and use case is provided.
```

Comment:
New feature requests are welcomed. Analyse whether the idea fits within the scope of project. If yes, detail your request while ensuring context and potential use cases.

Reply:
Then, detail your request, ensuring context and use case is provided.

I think this one needs update too. Multiple commas are not required.